src/ewatercycle/plugins/marrmot/forcing.py

```diff
@@ -112,3 +121,62 @@ def generate(  # type: ignore
 
 # TODO could be nice to have plot function
 # that loads the mat file into a xarray dataset and plots it
+
+
+# TODO make method of MarrmotForcing class? What to call it?
```

Comment:
Perhaps something like `to_xarray`? Something with "load" would be natural but it conflicts with the existing load method on the forcing class.